recipe/meta.yaml

```
@@ -23,28 +22,31 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - llvm-openmp    # [osx]
   host:
     # Host dependencies are pinned to last build
     - python
     - cython 0.29.33
-    - llvm-openmp 14.0.6  # [osx]
     - numpy   1.19   # [py<310]
```

```suggestion
    - numpy  {{ numpy }}
```

Please replace all numpy occurrences by the one from the cbc.